mlir/tools/mlir-clang/Lib/clang-mlir.cc

```diff
@@ -180,43 +180,143 @@ ValueWithOffsets MLIRScanner::VisitVarDecl(clang::VarDecl *decl) {
   return ValueWithOffsets(op, {zeroIndex});
 }
 
+bool MLIRScanner::getConstantLowerBound(clang::ForStmt *fors,
```

Comment:
Cool! just a quick comment: does this PR currently supports only constant upper/lower bounds?

Reply:
Yes.